astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,69 @@
+
+## The benefits of using an authentication service
+
+There are many benefits to using an authentication service. Authentication services can help you to:
+
+* Save time and money: Implementing and maintaining authentication can be time-consuming and expensive. Authentication services can save you the hassle of having to implement and maintain your own authentication system.
+* Improve security: Authentication services can help you to improve the security of your applications by using best practices and the latest security technologies keeping up to date with the latest security patches.
+* Reduce the risk of compliance violations: Authentication services can help you to meet compliance requirements by providing features such as two-factor authentication and user provisioning.
+
+The last thing that I want is a call on Saturday night that someone hacked our password database that we built in house and didn't secure!
+
+![Meme you can't leak passwords if you don't store passwords, man pointing at head](/img/blogs/i-just-want-authentication-to-work/meme-leak.png)
+
+## Choosing an authentication service
+
+When choosing an authentication service, there are a few factors to consider:
+
+* Features: Consider the features that are important to you, such as two-factor authentication, user provisioning, and support for different authentication methods.
+* Pricing: Authentication services can vary in price. Compare the pricing of different services to find the best deal for your needs.
+* Ease of use: Consider how easy the authentication service is to use. You want a service that is easy to set up and manage.
+* Hosting: Do you need flexibility to host your own instance of an Authentication Provider or have them Host.
+
+## For Developers Who Just Want It To Work
+
+If you are a developer who just wants authentication to work without implementing it yourself, there are a few things that you can do to make your life easier:
+
+* Use an authentication service. There are many different authentication services available, both free and paid. Authentication services can save you the hassle of having to implement and maintain your own authentication system.
+* Use a framework or library that provides authentication functionality. There are many different frameworks and libraries available that provide authentication functionality. Using a framework or library can save you the time and effort of having to implement authentication yourself. If you are ready to get started checkout our [FusionAuth Quickstarts](/docs/quickstarts/).
+* Consider that when Better Napster, wait I mean your app starts to grow, you will want an Authentication provider that can handle different forms of Authentication such as mobile, desktop and web.
+* Develop on a local instance or self contained envioronment that you can host yourself without risking data corruption in production.
+
+## What Should You Care About If Not Authentication
```

Comment:
Is this section new?  It doesn't really feel like this section belongs in this blog to me.  That could just be an opinion though.  I do think working the Auth as service video was good though.

Reply:
I wanted to get some backlinks pulling through. For me this is what the post is about. Once you have Auth figured out there is so much more to try and figure out for web now. SHould I tie it in better somehow saying that?

Reply:
Maybe

```suggestion
## What You _Should_ Care About
```